lib/reline/line_editor.rb

```diff
@@ -250,7 +250,8 @@ def reset_variables(prompt = '', encoding:)
     @resized = false
     @cache = {}
     @rendered_screen = RenderedScreen.new(base_y: 0, lines: [], cursor_y: 0)
-    @past_lines = []
+    @past_lines = [[[""], 0, 0]]
```

Comment:
Since we can now move through the past and future with undo and redo, the variable name `past_lines` doesn't seem very appropriate. As a suggestion, how about `input_lines`? If you have any other ideas, please let me know.

Reply:
Since I couldn't come up with a better name, I went with input_lines. Thank you.

[Renamed variables: past_lines -> input_lines](https://github.com/ruby/reline/pull/707/commits/0ab3c67be8ba157e3179bacccd2be14939fbe612)